packages/edit-site/src/components/page-pages/index.js

```diff
@@ -142,12 +150,37 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				filters: [ { id: 'author', type: 'enumeration' } ],
+				elements: [
+					{
+						value: '',
+						label: __( 'All' ),
+					},
+					...( authors?.map( ( { id, name } ) => ( {
+						value: id,
+						label: name,
+					} ) ) || [] ),
+				],
 			},
 			{
 				header: __( 'Status' ),
 				id: 'status',
 				accessorFn: ( page ) =>
 					postStatuses[ page.status ] ?? page.status,
+				filters: [ { type: 'enumeration', id: 'status' } ],
```

Comment:
I think the "id" here is a bit weird, it seems to just duplicate the field id for me.

Reply:
https://github.com/WordPress/gutenberg/pull/55440 addresses this.